jbpm-case-mgmt/jbpm-case-mgmt-impl/src/test/java/org/jbpm/casemgmt/impl/CaseServiceImplTest.java

```diff
@@ -876,6 +876,65 @@ public void testCaseWithComments() {
             }
         }
     }
+    
+    @Test
+    public void testCaseWithCommentsPagination() {
```

Comment:
need at all test case where no explicit offset and size are passed to queryContext (new QueryContext()) to make sure in that case all of case comments can be returned (not only 10). Otherwise imo this is a regression.